packaging/macos/dependencies-packaging.sh

```diff
@@ -32,4 +32,4 @@ brew install \
 # Install dependencies with pip
 pip3 install \
   pygobject \
-  pyinstaller==4.1
+  git+https://github.com/pyinstaller/pyinstaller@da88bc85211befe10c4090a880db1a4c8d3ea625
```

Comment:
Using a development version for now, since https://github.com/pyinstaller/pyinstaller/commit/5ae5a2d4af15e570a74bc4541ac153ad12cddec5 is necessary.